src/ontology/mondo-ingest.Makefile

```diff
@@ -181,6 +182,8 @@ config/%_term_exclusions.txt: config/%_exclusions.tsv component-download-%.owl $
 	--component-signature-path $(word 4,$^) \
 	--config-path $(word 5,$^)
 
+all-exclusions: config/ordo_term_exclusions.txt config/doid_term_exclusions.txt config/omim_term_exclusions.txt config/icd10cm_term_exclusions.txt config/icd10who_term_exclusions.txt config/ncit_term_exclusions.txt
```

Comment:
@matentzn It'd be nice to have some tests.

I also just realized we have no `all` command. We usually want that, right? That could sort of serve as a broad end-to-end test.

It could be good to have an automated build on GitHub using `make all`. But I suppose the heavy memory burden of he docker container, especially for NCIT, could be a problem for that.

Reply:
`make all` does not make too much sense for ODK as we have multiple disconnected pipelines (release, matching, exclusion tables) in there - it would be arbitrary to say that `release` is the `all` im my opinion, but could be convinced this is clearer.

Reply:
Fine with me